### ISSUE TITLE
The label does not correspond to the box

### ISSUE THREAD

**Issue by Finorita** [Megvii-BaseDetection/YOLOX#1328]
My training process refers to the Train Custom Data on the official website .
Instead of the example, I used COCO format dataset, which is about 500 images labeled boat.
In this process, I only changed self.numclasses=80 directly on exps/custom/yolox_s.py.
(An error is reported if it equals 1.)
The categories item of my JSON file is consistent with the coco dataset. Categories id = 9.

Here are the test results of my training model. (fine-tuning)( use demo.py & configuration file used during training---yolox_s.py）

![dog3](https://user-images.githubusercontent.com/105027890/169034613-198f9aff-4d3e-4ba5-8ed8-217e826a81bc.jpg)
![dog  2](https://user-images.githubusercontent.com/105027890/169034763-330e42f3-0cbc-4913-8109-6ed19631d40e.jpg)

befor fine-tuning
![dog  11](https://user-images.githubusercontent.com/105027890/169035732-e0a842f9-a052-4861-bd5f-80f9bfa988ed.jpg)
![dog2  2](https://user-images.githubusercontent.com/105027890/169035783-760aba37-c3b6-4f2a-a7f6-63cd8e0b9d1c.jpg)
![dog3  2](https://user-images.githubusercontent.com/105027890/169035828-ca76e478-7492-40c2-968f-de97a969bec8.jpg)

**Comment by Finorita** [comment]
你好，你的邮件我已收到。感谢你的来信。

**Comment by Finorita** [comment]
![7I3`{8S71UK)2~%KW3PVNJY](https://user-images.githubusercontent.com/105027890/169236876-522841a9-898f-40db-989a-6b5cf3065669.png)
**↑↑ my model Effect on my test set**
**↓↓ my model Effect on COCO2017 （val2017）**
![8DS7J__36LYSDS)$TWMZ{ V](https://user-images.githubusercontent.com/105027890/169237153-9d4fac05-3455-4cf0-b497-17e2b5c3d186.png)

**Comment by Finorita** [comment]
yolox.pth on coco2017(val2017)
![X 4BD%L@ZS)XJC`RMZ)}IOD](https://user-images.githubusercontent.com/105027890/169237487-8c1cafa8-d350-4cb6-94d5-2254a67970d6.png)
![{IYK4V2TA4~IF(Z{XMQ2TZN](https://user-images.githubusercontent.com/105027890/169237547-739145de-ac04-4270-a041-1d519c4cac47.png)

**Comment by Finorita** [comment]
> yolox.pth on coco2017(val2017)

yolox.pth on coco2017(val2017) and my test sets

**Comment by Finorita** [comment]
OMG!!!!!  Because I didn't even open resume! Fine tuning was performed only when Resume =True, and I also found errors in the remarks of resume files should be wrong.I don't know if I understand right? Thank you for your attention!
![image](https://user-images.githubusercontent.com/105027890/169500276-766beaed-7519-44f7-807d-e44d179d9052.png)

**Comment by Finorita** [comment]
> Since you are finetuning YOLOX on customed dataset, did you evalutate your model on it?

OMG!!!!! Because I didn't even open resume! Fine tuning was performed only when Resume =True, and I also found errors in the remarks of resume files should be wrong.I don't know if I understand right? Thank you for your attention!

**Comment by Finorita** [comment]
这英语我实在懒得翻译了。最后一次更新了！！！算是给后来者的一个提醒吧！之前出现这个情况我可以确定就是过拟合了！应该需要进行fine turn。以下是我对这个resume 的小结！！！
![image](https://user-images.githubusercontent.com/105027890/169809444-82a85a06-4540-4011-a5e5-c97d2e7db009.png)

**Comment by Finorita** [comment]
It's not overfitting, it's data mismatch. It is not possible to increase the ability to recognize one category while maintaining the ability to recognize others through fine-tuning alone.
